src/explorer/trialApp/TrialAppTreeItem.ts

```diff
@@ -95,6 +98,16 @@ export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem
     public isAncestorOfImpl?(contextValue: string | RegExp): boolean {
         return contextValue === TrialAppTreeItem.contextValue;
     }
+
+    public compareChildrenImpl(item1: AzExtTreeItem, item2: AzExtTreeItem): number {
+        if (item1 instanceof GenericTreeItem) {
+            return -1;
+        }
+        if (item2 instanceof GenericTreeItem) {
+            return 1; // tutorial node at top
```

Comment:
nit: Can you move the comment up above line 103 so it's more obvious both checks are for the tutorial node?